examples/plotting/file/sprint.py

```diff
@@ -54,7 +58,7 @@ def selected_name(name, medal, year):
 
 source = ColumnDataSource(sprint)
 
-xdr = Range1d(start=sprint.MetersBack.max()+2, end=0)               # XXX: +2 is poor-man's padding (otherwise misses last tick)
+xdr = Range1d(start=sprint.MetersBack.max()+2, end=0)      # XXX: +2 is poor-man's padding (otherwise misses last tick)
 ydr = DataRange1d(range_padding=4, range_padding_units="absolute")
 
 plot = figure(
```

Comment:
We could also dispense with the explicit range imports, something like
```
plot = figure(y_range=(sprint.MetersBack.max()+2, end=0), ...)
plot.x_range.range_padding = 4
plot.x_range.range_padding_units = "absolute"
```
What do you think?

I guess all things being equal, I feel like examples are often more approachable with fewer imports.